runtime/lua/nvim/_fix_list.lua

```diff
@@ -0,0 +1,59 @@
+-- TODO(tjdevries): Make the metamethods work for __len
+--                      I can't seem to get that to work the way I want it to.
+local FixList = {}
+
+FixList.new = function(self, set, get, open, close, title)
```

Comment:
This module should be deleted. We should not add tons of new Lua modules that are just thin wrappers around VimL functions. Call the VImL functions directly via `nvim_call_function` (and yes, we should make it easier to call VimL functions :)

Reply:
I deleted it because it was in the original PR and probably not used now.